python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -3161,6 +3163,103 @@ def add_backfill(self, partition_backfill: "PartitionBackfill") -> None:
     def update_backfill(self, partition_backfill: "PartitionBackfill") -> None:
         self._run_storage.update_backfill(partition_backfill)
 
+    @public
+    def launch_backfill(
+        self,
+        asset_graph: "BaseAssetGraph",
+        asset_selection: Optional[Sequence[AssetKey]] = None,
+        partitions_by_assets: Optional[
+            Sequence[Tuple[AssetKey, Optional[Sequence[Tuple[str, str]]]]]
+        ] = None,
```

Comment:
This avoids exposing any additional objects, but I suppose an explanation/examples will be necessary for anybody unfamiliar with Dagster internals already.